modules/jar-handling-utils/src/main/java/org/mule/runtime/module/artifact/api/classloader/jar/CachingURLStreamHandlerFactory.java

```diff
@@ -17,7 +17,7 @@
 public class CachingURLStreamHandlerFactory {
 
   public static URLStreamHandlerFactory getCachingURLStreamHandlerFactory() {
-    return new NonCachingURLStreamHandlerFactory();
+    return null;
```

Comment:
We should remove this and just use `null` where appropriate I think.

Reply:
proper cleanup will be tackled separately